### DISCUSSION TITLE
Cron NOT working

### DISCUSSION THREAD

**Discussion by legioz** [deta/cloud-docs#250] [Help]
I am using using deta **1.1.0** with Python with the exactly same code in the [docs](https://docs.deta.sh/docs/micros/cron/#cron-and-http)
Every time I try to run the code I get this error: ImportError: cannot import name 'App' from 'deta'

```python
from deta import App
from fastapi import FastAPI

app = App(FastAPI())

@app.get("/")
def http():
    return "Hello Deta, I am running with HTTP"

@app.lib.cron()
def cron_job(event):
    return "Hello Deta, I am a cron job"
```
---
I checked the deta code in deta/__init__.py
This piece of code in deta lib is not working:
    
![image](https://user-images.githubusercontent.com/40257388/172073297-3951e03e-28aa-44df-bfe3-0d9217fc81ea.png)

The exception raised is **No module named 'detalib'**

**Comment by abdelhai** [comment]
hi, this is a feature of deta micros, as such the code has to be deployed on a micro to work. 
we do not yet offer a local-testing capability for cron

**Reply by gkuznik** [reply]
I have deployed this code in a micro and still get this error: `ImportError: cannot import name 'App' from 'deta' (/var/task/deta/__init__.py)
`

**Reply by abdelhai** [reply]
do you have a directory/folder in your code named `deta`? if so rename it to something else. it looks like an import conflict

**Reply by gkuznik** [reply]
Yes, I had one.

**Comment by lemonyte** [comment]
Duplicate of #402.
